packages/lib/utils/tests/chunkGenerator.test.ts

```diff
@@ -0,0 +1,45 @@
+import chunkGenerator from '../chunkGenerator';
+import * as fs from 'fs';
```

Comment:
Only import the functions that you need and only use `fs/promise`. Please have a look at how we use `fs/promise` in other files.